spinoso-time/src/time/tzrs/offset.rs

```diff
@@ -100,38 +108,41 @@ impl<'a> Offset {
         let offset_name = offset_hhmm_from_seconds(offset);
         let local_time_type =
             LocalTimeType::new(offset, false, Some(offset_name.as_bytes())).expect("Couldn't create fixed offset");
-        Self::Fixed([local_time_type])
+        Self {
+            inner: OffsetType::Fixed(local_time_type),
+        }
     }
 
     /// Generate an offset based on a provided [`tz::timezone::TimeZoneRef`].
     ///
     /// This can be combined with [`tzdb`] to generate offsets based on
     /// predefined IANA time zones.
     ///
-    /// ```
-    /// use spinoso_time::tzrs::Offset;
-    /// use tzdb::time_zone::pacific::AUCKLAND;
-    /// let offset = Offset::tz(AUCKLAND);
-    /// ```
+    /// [`tz:timezone::TimeZoneRef`]: https://docs.rs/tz-rs/0.6.9/tz/timezone/struct.TimeZoneRef.html
+    /// [`tzdb`]: https://docs.rs/tzdb/latest/tzdb/index.html
```

Comment:
these comments are not necessary. rustdoc will resolve these links to the actual docs built.

Reply:
Must have come in with a merge resolution and I thought they were meant to be there. whoops, removing